README.md

```diff
@@ -1,2 +1,49 @@
-# unsere-app-fur-die-universitat-regensburg-code-companion
-unsere-app-fur-die-universitat-regensburg-code-companion created by GitHub Classroom
\ No newline at end of file
+# Code Companion
+
+## Concept
+The Goal of the Project is to provide realtime coding assistance on a second screen to help beginners with their coding tasks.  
+This is implemented using an Android App providing real time compiler errors, tasks and coding statistics as well as providing notifications for upcoming project deadlines.
+
+## Implementation
+
+The Project consists of a Plugin for the Jetbrains IntelliJ Platform (plugin marketplace link coming soon) reporting to the connecting Android App (google play link coming soon) using WebRTC.
+This App shows the current tasks, compiler errors and coding statistics.  
+
+
+![](Screenshot.png)  
+
+
+## Team
+Code Companion is developed by Fabian (@PLACEBOBRO) Lukas (@RealWhimsy) Mathias (@goetzmat) and Maximilian (@MaximilianSeewald) during the "Advanced Software Engineering" Course of Uni Regensburg.
+
+
+## Template Project
+
+A template project can be found -here-.
+
+## Task Information
+
+The "task.json" file is located inside the root of the project folder, and holds the project name, tasks and project due date.  
```

Comment:
Put this as a sub-sub-section of the **Task Fragment** overview. Make clear what the fields of the tasks.json do ("tasks" array is what is displayed in the app etc.) and explain what happens when no tasks.json is present